www/grpc/server.go

```diff
@@ -80,7 +81,9 @@ func (s *Server) startListening(listener net.Listener) error {
 
 	if s.config.EnableWallet {
 		chainType := s.state.Genesis().ChainType()
-		walletServer := newWalletServer(s, chainType)
+
+		service := wallet.NewWalletService(chainType)
```

Comment:
Please use Dependency Injection patterns here.
Let's create a Service (Manager) in `node.NewNode` and pass it here.